schedule/virt_autotest/tumbleweed-virt-guest-install.yaml

```diff
@@ -43,26 +42,29 @@ conditional_schedule:
         SKIP_GUEST_INSTALL:
             0:
                 - virt_autotest/unified_guest_installation
+                - virt_autotest/set_config_as_glue
     virtual_network_test:
```

Comment:
I think even with `SKIP_GUEST_INSTALL=1`,  `virt_autotest/set_config_as_glue` is still needed, because `set_config_as_glue` saves the guest data to a hash, without it the subsequent tests are unable to run. Are you able to run virtual network tests without `virt_autotest/set_config_as_glue`?

Reply:
Actually, you are right the schedule and test suit settings should be in sync
![image](https://github.com/os-autoinst/os-autoinst-distri-opensuse/assets/46835794/d3b4b6a9-e263-4198-85c5-372afb135c9d)
it was set to zero and it is enabled and it shows in test 
so virt_autotest/set_config_as_glue is always in test runs 
https://openqa.opensuse.org/tests/3592782#

Reply:
when there is a running guest, you set `SKIP_GUEST_INSTALL=1` to skip installing a fresh guest. It is helpful when you debug feature tests, you need not install a new guest repeatedly in your tests. fg. Use `DO_NOT_INSTALL_HOST=1 SKIP_GUEST_INSTALL=1 IPMI_DO_NOT_POWER_OFF=1 IPMI_DO_NOT_RESTART_HOST=1 ENABLE_VIR_NET=1` to run virtual network tests only with a running TW guest on a TW host. It saves effort.

Reply:
ok, thanks